public/noscript.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: normal;
+    font-weight: 700;
+    src: url("/fonts/gotham-rounded-bold.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: italic;
+    font-weight: 700;
+    src: url("/fonts/gotham-rounded-bold-italic.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: normal;
+    font-weight: 400;
+    src: url("/fonts/gotham-rounded-book.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: italic;
+    font-weight: 400;
+    src: url("/fonts/gotham-rounded-book-italic.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: normal;
+    font-weight: 300;
+    src: url("/fonts/gotham-rounded-light.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: italic;
+    font-weight: 300;
+    src: url("/fonts/gotham-rounded-light-italic.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: normal;
+    font-weight: 500;
+    src: url("/fonts/gotham-rounded-medium.otf");
+}
+
+@font-face {
+    font-family: "Gotham Rounded";
+    font-style: italic;
+    font-weight: 500;
+    src: url("/fonts/gotham-rounded-medium-italic.otf");
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+noscript {
+    font-family: Gotham Rounded;
+    font-style: normal;
+    font-size: 14px;
+    display: flex;
+    justify-content: center;
+    background: #E5E5E5;
```

Comment:
@RodrigoLeiteF just quick checked Figma and the color for the background is #F8F8F8 (https://www.figma.com/file/9BX6vISFLh3Ns0HtVx2XJX/Special-pages?node-id=22%3A1059)